packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -65,50 +67,51 @@ export default class QuanticSearchBoxInput extends LightningElement {
    */
   @api suggestions = [];
   /**
-   * Returns and set the input value.
+   * The value of the input.
    * @api
-   * @type {string}
+   * @type {String}
    */
-  @api
-  get inputValue() {
-    return this.input.value;
-  }
-  set inputValue(newValue) {
-    this.input.value = newValue;
-  }
+  @api value;
   /**
-   * The blur function.
+   * The list containing the recent query suggestions.
    * @api
-   * @type {VoidFunction}
+   * @type {String[]}
    */
-  @api blur() {
```

Comment:
Don't delete it just for fun.
If possible still make it work to blur the input when this is called.
And mark it by a comment that it's `@deprecated` and create a jira to remove it in v3.